modules/.eslintrc.js

```diff
@@ -85,7 +85,7 @@ const config = {
 		'notice/notice': [
 			'error',
 			{
-				nonMatchingTolerance: 0.7,
+				nonMatchingTolerance: 0.95,
```

Comment:
Why not 100%? :thinking:

Reply:
There are weird issues with trailing new lines and it would add repeated new lines at the end of the header.